modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -28,10 +28,189 @@ If you have installed the {gitops-title} Operator in the default namespace, run
+
+* Before this update, when the desired state for the server service type changes, for example, from `ClusterIP` to `NodePort`, the change was not applied automatically to the cluster without first deleting the resource. This update fixes the issue by adding a server service check to apply changes to the server service. If the desired state for the server service type changes, it is applied automatically to the cluster without deleting the resource. link:https://issues.redhat.com/browse/GITOPS-5385[GITOPS-5385]
+
+* Before this update, users were unable to set a custom image for the `ApplicationSet` Controller after specifying it in the Argo CD CR. This update fixes the issue by allowing users to set a custom image for the `ApplicationSet` Controller. link:https://issues.redhat.com/browse/GITOPS-5395[GITOPS-5395]
+
+* Before this update, Argo CD was unable to retrieve the correct TLS certificate for Helm OCI registries if the URL contained a path or port number. This update fixes the issue by using a fix in upstream Argo CD that ensures the URL is correctly parsed, allowing Argo CD to retrieve and return a valid TLS certificate. link:https://issues.redhat.com/browse/GITOPS-5081[GITOPS-5081]
+
+[id="known-issues-1-15-0_{context}"]
+== Known Issues
+
+* There is currently a known issue that causes decoupling application synchronization using impersonation to fail due to insufficient permissions in the `openshift-gitops-openshift-gitops-argocd-application-controller` `ClusterRole`. This `ClusterRole` lacks the necessary permissions required for impersonation, leading to a failure during application synchronization. When encountering this issue, you may see an error message similar to the following:
++
+.Example
+[source,YAML]
+----
+serviceaccounts "<serviceaccount name>" is forbidden: User "system:serviceaccount:openshift-gitops:openshift-gitops-argocd-application-controller" cannot impersonate resource "serviceaccounts" in API group "" in the namespace "<destination namespace>"
+----
++
+Workaround: To address this issue:
++
+* Manually edit the `openshift-gitops-openshift-gitops-argocd-application-controller` ClusterRole to include the necessary permissions for impersonation.
+* Add `ClusteRole` and `ClusterRolebBnding` for providing the required permission after enabling impersonation.
+** Create a new `ClusterRole` with impersonate permission.
++
+.Example
+[source,YAML]
+----
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: impersonation
+rules:
+  - verbs:
+      - impersonate
+    apiGroups:
+      - ''
+    resources:
+      - serviceaccounts
+----
+** Add a new `ClusterRole` to Argo CD Application Controller service account using a new `ClusterRoleBinding` role.
```

Comment:
```suggestion
** Add a new `ClusterRole` to `openshift-gitops-argocd-application-controller` service account using a new `ClusterRoleBinding` role.
```

Reply:
Fixed. @varshab1210 and @anandf have recommended removing this feature, so I am removing the known issue content.